src/Dev/Deprecation.php

```diff
@@ -353,4 +451,29 @@ public static function restore_settings($settings)
         static::notice('4.12.0', 'Will be removed without equivalent functionality to replace it');
         // noop
     }
+
+    private static function varAsBoolean($val)
```

Comment:
```suggestion
    /**
     * @var mixed $val
     */
    private static function varAsBoolean($val): bool
```

Reply:
I don't think that provides any value. Mixed is implied.

Reply:
True, should add the return type though

Reply:
Done